web_widget_float_formula/static/src/js/web_widget_float_formula.js

```diff
@@ -52,12 +53,7 @@ odoo.define('web_widget_float_formula', function(require) {
             },
 
             _process_formula: function(formula) {
-                try{
```

Comment:
Oops, I must have accidentally deleted that with my console logs. Putting it back.